mkdocs.yml

```diff
@@ -86,67 +84,23 @@ nav:
-                      - microservices/device/services/device-onvif-camera/supplementary-info/utility-scripts.md
-                  - Miscellaneous:
-                      - microservices/device/services/device-onvif-camera/supplementary-info/control-plane-events.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/getting-started-with-docker-security.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/ws-discovery.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/onvif-footnotes.md
-          - Device REST: microservices/device/services/device-rest.md
-          - Device RFID LLRP: microservices/device/services/device-rfid-llrp.md
-          - Device SNMP: microservices/device/services/device-snmp.md
-          - Device UART: microservices/device/services/device-uart.md
-          - Device USB Camera:
-              - microservices/device/services/device-usb-camera/General.md
-              - Walkthrough:
-                  - microservices/device/services/device-usb-camera/walkthrough/setup.md
-                  - microservices/device/services/device-usb-camera/walkthrough/deployment.md
-                  - microservices/device/services/device-usb-camera/walkthrough/general-usage.md
-                  - microservices/device/services/device-usb-camera/walkthrough/custom-build.md
-              - Supplementary Information:
-                  - microservices/device/services/device-usb-camera/supplementary-info/USB-protocol.md
-                  - microservices/device/services/device-usb-camera/supplementary-info/discovery.md
-                  - microservices/device/services/device-usb-camera/supplementary-info/advanced-options.md
-          - Device Virtual: microservices/device/services/device-virtual/Ch-VirtualDevice.md
-          - Remote Deployment:
-              - Non-Secure Mode: microservices/device/remote/Ch-RemoteNonSecure.md
-              - Secure Mode: microservices/device/remote/Ch-RemoteSecure.md
+        - Device Service: microservices/device/DeviceService.md
+        - Getting Started: microservices/device/GettingStarted.md
+        - Configuration: microservices/device/Configuration.md
+        - API Reference: microservices/device/ApiReference.md
+        - Device Service SDK:
+            - Purpose: microservices/device/sdk/Purpose.md
+            - Getting Started: 
+                - GettingStartedSDK-C: microservices/device/sdk/devicesdk-getting-started/GettingStartedSDK-C.md
+                - GettingStartedSDK-Go: microservices/device/sdk/devicesdk-getting-started/GettingStartedSDK-Go.md
+            - APIs:
+                - C Device SDK API: microservices/device/sdk/api/CDeviceSDKAPI.md            
+                - Go Device SDK API: microservices/device/sdk/api/GoDeviceSDKAPI.md
+            - Source Code: microservices/device/sdk/SourceCode.md
```

Comment:
Please create `sdk Additional Details" and added the following:
- CustomConfiguration.md - Similar to https://github.com/edgexfoundry/edgex-docs/blob/main/docs_src/microservices/application/sdk/details/CustomConfiguration.md
- CustomRestApis.md - similar to https://github.com/edgexfoundry/edgex-docs/blob/main/docs_src/microservices/application/sdk/details/CustomRestApis.md
- CustomServiceMetrics.md - similar to https://github.com/edgexfoundry/edgex-docs/blob/main/docs_src/microservices/application/sdk/details/CustomServiceMetrics.md
- Secrets.md - similar to https://github.com/edgexfoundry/edgex-docs/blob/main/docs_src/microservices/application/sdk/details/Secrets.md